frontend/components/common/Login.tsx

```diff
@@ -5,27 +5,44 @@ function Login() {
   return (
     <div>
       <div
+        className="inner-home"

```

Comment:
where is inner-home implemented?

Reply:
I was trying to use custom css but I guess it was not required. Hence I will remove that.